generated/nidmm/session.py

```diff
@@ -2327,7 +2328,7 @@ def get_measurement_period(self):
         period_ctype = ctypes_types.ViReal64_ctype(0)
         error_code = self.library.niDMM_GetMeasurementPeriod(self.vi, ctypes.pointer(period_ctype))
         errors._handle_error(self, error_code)
-        return period_ctype.value
+        return python_types.ViReal64(period_ctype.value)
```

Comment:
Were these changes intentional? I don't fully understand the implications.
We are now returning an object of a different type than we used to, and in the case of booleans we aren't really returning True/False.

To clients, is this?
* Transparent
* Helpful
* Detrimental
And how?
(mostly I want to understand it)